### PR TITLE
Current progress on frontend 

### PR DESCRIPTION
**Pull request by aashnasheth**
Current progress on frontend with dummy data

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,104 +1,23 @@
...
-typings/
-
-# TypeScript cache
-*.tsbuildinfo
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
-# Optional npm cache directory
-.npm
+# dependencies
+/node_modules
```
Comment (kushaljh) [inline .gitignore:4]: I think this should be node_modules/ and not /node_modules

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```
Comment (kushaljh) [inline .idea/.gitignore:1]: Please add .idea/ in your root .gitignore
You do not need this .gitignore as the whole .idea directory is a cache and does not need to be on git.

File: src/connect.js [hunk 3]
```diff
@@ -1,17 +0,0 @@
-let mysql = require('mysql');
```
Comment (kushaljh) [inline src/connect.js:1]: I am not sure if we want to delete this

Can @TylerDillon, @daniellee0, or @wilsmic2 confirm if this file is needed?

File: src/query.js [hunk 4]
```diff
@@ -1,32 +0,0 @@
-let mysql = require('mysql');
```
Comment (kushaljh) [inline src/query.js:1]: Same for this.

File: src/connect.js (hunk 3 above)
Comment (daniellee0) [reply src/connect.js:1]: This file isn't needed as the server already has JS to establish the connection instead (see server.js and routes.js).

File: .gitignore (hunk 1 above)
Comment (aashnasheth) [reply .gitignore:4]: I think when you link a react project to git, /node_modules is automatically added to git ignore.
isn't /node_modules the path down to the modules folder from dashboard? is there a reason you think node_modules/ would be better?

Comment (kushaljh) [reply .gitignore:4]: You do not need to go to /node_modules
As that would go to root and then the node_modules.

**Pull request merged**